### ISSUE TITLE
Error trying to parse settings

### ISSUE THREAD

**Issue by treechime** [akalongman/sublimetext-codeformatter#152]
When installing from Package Control I get this message:

```
Error trying to parse settings: Trailing comma before closing bracket in C:\Users\user\AppData\Roaming\Sublime Text 2\Packages\CodeFormatter\CodeFormatter.sublime-settings:51:2
```

![image](https://cloud.githubusercontent.com/assets/8661599/8841148/88861262-312d-11e5-8880-ceed05ac96d2.png)

From then on if I try to format code I get an error saying:

```
CodeFormatter
Format error:
argument of type 'NoneType' is not iterable
```

![image](https://cloud.githubusercontent.com/assets/8661599/8841165/b27e6452-312d-11e5-9b3d-570341831006.png)

Which is also mentioned here

https://github.com/akalongman/sublimetext-codeformatter/issues/151

Removing the comma on this line in CodeFormatter.sublime-settings:51:2 (ironically about commas) fixes the problem
`"comma_first": false, // Add comma first`

**Comment by jigsawIV** [comment]
Thanks @treechime.
I removed the comma and it worked.

**Comment by akalongman** [comment]
Fixed in 2.6.0 version
